bedrock/firefox/urls.py

```diff
@@ -84,9 +84,6 @@
     page('firefox/private-browsing', 'firefox/private-browsing.html', alt_canonical_paths={
```

Comment:
Seeing as we're removing most of the other legacy pages in this PR, can `firefox/private-browsing` be retired as well?